otel-agent/k8s-helm/values.yaml

```diff
@@ -48,6 +49,10 @@ opentelemetry-collector:
       enabled: true
     kubeletMetrics:
       enabled: true
+    metadata:
+      enabled: true
+      clusterName: "{{.Values.global.clusterName}}"
+      integrationName: "coralogix-otel-agent-helm"
```

Comment:
We use this parameter / attribute in `otel-integration` to identify metrics for the K8s dashboard. Does the same apply for this chart? Or is there other reason to introduce it?

Reply:
Not strictly against, I'm just thinking if we have a good enough reason to have this at the moment, as for `otel-integration` it is required for proper functioning of dashboards. Just trying to decrease the number of attributes, to keep cardinality / costs low for users, given this will be attached to every metric.

Reply:
Maybe not directly but I still think it's a good rule to try keep number of labels to minimum, unless we do believe these are useful. Let's keep it then if we see value in having them.